wire-ios-mocktransport/Source/MockTransportSession.m

```diff
@@ -490,7 +490,10 @@ - (void)completeRequest:(ZMTransportRequest *)request completionHandler:(ZMCompl
         NSAssert(pair.count == 2, @"Unexpected pair count in method map");
         NSString *path = pair[0];
         NSString *selectorString = pair[1];
-        if ([request.path hasPrefix:path])
+
+
+        NSString* requestPathWithoutVersion = [request.path removingAPIVersion];
```

Comment:
might be better to store the apiVersion has a property of MockTransport so we can have different implementations per path...